permissions-policy-explainer.md

```diff
@@ -171,14 +167,14 @@ describes some of these questions in more detail.
 
 ## Permissions Policy and the Reporting API
 
-Permissions policy integrates with the Reporting API, so that you can get reports 
+Permissions Policy integrates with the Reporting API, so that you can get reports 
 from users when a policy is violated in their browser, or respond to violations 
-in JavaScript. See the separate explainer, [Feature Policy Reporting](reporting.md), for 
+in JavaScript. See the separate explainer, [Permissions Policy Reporting](reporting.md), for 
```

Comment:
I didn't update this, since the other document's title hadn't changed yet.

Reply:
Yeah, I thought about that. But I figured might as well update this now, because it seemed likely to be forgotten. Also, maybe a forcing function to update the other doc soon.

Reply:
Definitely; I'll take a look at that doc soon. There's also #386 to work out as well, but that shouldn't stop this from landing.